### DISCUSSION TITLE
feat (ui): Collapse sidebar by default

### DISCUSSION THREAD

**Discussion by marctorsoc** [langfuse/langfuse#2881] [Ideas]
### Describe the feature or potential improvement

I see that when the window is small enough, the sidebar is collapsed to allow for more space for the data. I wonder if we should have the sidebar permanently collapsed. It is quite natural to click on the 3 lines icon if you want to find something and for big screens we want see all columns for traces more clearly. WDYT? 

### Additional information

An alternative is to add a button to collapse as per user's request

**Comment by marcklingen** [comment]
agree, this would be super useful. I am wondering how to best collapse the menu when there are second-level menu items.

![CleanShot 2024-08-13 at 16 33 53](https://github.com/user-attachments/assets/54dfa34c-5f9d-422b-9dc6-106cfc2bfb31)

**Comment by marctorsoc** [comment]
It already gets collapsed, from
![image](https://github.com/user-attachments/assets/58270443-7eb4-4fb4-a1fb-d7425e81463b)
to 
![image](https://github.com/user-attachments/assets/88b12158-1348-4a7f-a8a5-650fc7dadf18)

and then when clicking
![image](https://github.com/user-attachments/assets/8a9e97b4-5bf7-4961-a522-98b4b4aff955)

My proposal is to have the latter behaviour as the default, not only when the screen is narrow

**Comment by marctorsoc** [comment]
Initial implementation here: https://github.com/langfuse/langfuse/pull/3005  @marcklingen 

lmk your thoughts. I wonder if we want this to be configurable. Otherwise, it was a quite simple change. Just removing stuff 😄

**Reply by marcklingen** [reply]
Thanks for the contribution! sorry for the slow response here.

I agree that defaulting to the mobile/collapsed menu on desktop screens is an option. However this makes it harder than necessary to switch between features and discover them when new to the product. Most tools I use daily show the menu on larger screens but allow to collapse it. I added this issue as i'd love to support the implementation of it: https://github.com/langfuse/langfuse/issues/3501

**Reply by marcklingen** [reply]
What do you think @marctorsoc?

**Reply by marctorsoc** [reply]
sure, sounds good to me. I think it'd be nice to have something in settings to remember your preferences. But that can be future feature

**Comment by marcklingen** [comment]
Tracking this here: https://github.com/langfuse/langfuse/issues/3501

I think a fully collapsed menu on desktop makes discovering and navigating between features unnecessarily difficult, being able to change the width of the menu is a better implementation

**Comment by marctorsoc** [comment]
Closing since there's already an issue to track
